gui/cefdisplay/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+############################################################################
+# CMakeLists.txt file for building ROOT gui/canvaspainter package
+############################################################################
+
+if(DEFINED ENV{CEF_PATH})
+if(EXISTS $ENV{CEF_PATH})
```

Comment:
I'd prefer to configure this once and then store it, as in
```
set (CEF_PATH $ENV{CEF_PATH})
```
Then only `add_directory` if this is set. But @amadio should overrule me here.

Reply:
I just copy my temporary cmake code here.
One should create FindCEF.cmake script, where different CEF files should be tested
and configured directory stored in CMAKE files. But I have no experience writing such files.

Reply:
Apparently they already provide a usable `FindCEF.cmake`: https://bitbucket.org/chromiumembedded/cef/src/ac86b611391b606170351bae322be1dd18d5a38b/CMakeLists.txt.in?at=master&fileviewer=file-view-default#CMakeLists.txt.in-187

Reply:
> Apparently they already provide a usable FindCEF.cmake

But it only checks CEF_ROOT variable (which probably we could use as well).
One should add few more checks - at the best for all files which could be required for compilation and starting CEF

Reply:
> Apparently they already provide a usable FindCEF.cmake

@amadio 

Hi Guilherme
Another question - could we easily use it inside ROOT cmake.
As a base, we are using cefsimple example application.
Here is cmake file for it. 

https://bitbucket.org/chromiumembedded/cef/src/ac86b611391b606170351bae322be1dd18d5a38b/tests/cefsimple/CMakeLists.txt.in?at=master&fileviewer=file-view-default

Could we use it in such form in the ROOT?
Main difference - instead of cefsimple executable libROOTCefDisplay library is produced, 
which is also linked with ROOT libraries.
And instead of sandbox application cef_main.cxx is created.

Reply:
The file linked above is meant to be processed by some tool before being used. Probably something from CEF itself. With some adaptation, a similar thing to the file above could be used in ROOT.